did/cli.py

```diff
@@ -218,7 +218,7 @@ def main(arguments=None):
             if options.merge:
                 utils.item(user, 1, options=options)
             else:
-                utils.header(user)
+                utils.header(user, did.base.Config().border_count, did.base.Config().border_char)
```

Comment:
Shouldn't the same format be used for the `Total Report` header as well?

Reply:
Added

Reply:
There's actually one more instance of the `header()` on line 213.

Reply:
Got it